website/project/views/node.py

```diff
@@ -399,6 +399,26 @@ def configure_comments(node, **kwargs):
 def configure_requests(node, **kwargs):
     access_requests_enabled = request.get_json().get('accessRequestsEnabled')
     node.access_requests_enabled = access_requests_enabled
+    if node.access_requests_enabled:
+        node.add_log(
```

Comment:
Drive-by: This is OK for now, but logging should really go on the model rather than the view. This makes testing easier, and it will make it easier to write the v2 code.

Reply:
Made a ticket: https://openscience.atlassian.net/browse/PLAT-824